rpm_spec/subpackages/manageiq-gemset-services

```diff
@@ -0,0 +1,34 @@
+%package gemset-services
+Summary: %{product_summary} Gemset Services
+
+Requires: %{name}-gemset = %{version}-%{release}
+
+%description gemset-services
+%{product_summary} Gemset Services
+
+%post gemset-services
+# Remove any old systemd unit files
+provider_worker_types="event_catcher metrics_collector refresh operations"
+provider_types="amazon_cloud_manager ansible_tower_automation_manager azure_cloud_manager autosde_storage_manager azure_stack_cloud_manager azure_stack_network_manager foreman_configuration_manager google_cloud_manager google_network_manager ibm_cloud_vpc_cloud_manager ibm_cloud_power_virtual_servers_cloud_manager ibm_terraform_configuration_manager kubernetes_container_manager kubernetes_monitoring_manager kubevirt_infra_manager lenovo_physical_infra_manager microsoft_infra_manager nsxt_network_manager nuage_network_manager openshift_container_manager openshift_monitoring_manager openstack_cloud_manager openstack_infra_manager openstack_network_manager openstack_storage_manager_cinder_manager amazon_storage_manager_s3 redhat_infra_manager redhat_network_manager redfish_physical_infra_manager vmware_cloud_manager vmware_infra_manager"
```

Comment:
😕  This is going to kill pluggability

Reply:
I wonder if the plugins can bring an rpm_spec directory which we can pull in some sort of partial from.

Reply:
This is only to remove old unit files, so it doesn't impact pluggability going forward.  Technically we only need to do this cleanup once.